website/docs/r/sentinel_alert_rule_anomaly_built_in.html.markdown

```diff
@@ -0,0 +1,172 @@
+---
+subcategory: "Sentinel"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_sentinel_alert_rule_anomaly_built_in"
+description: |-
+  Manages a Built-in Anomaly Alert Rule.
+---
+## Disclaimers
+
+~> **Note:** A Built-in Anomaly Alert Rule could not be deleted. delete a Terraform managed Built-in Anomaly Alert Rule will cause the Built-in Anomaly Alert Rule to be disabled.
+
+# azurerm_sentinel_alert_rule_anomaly_built_in
+
+Manages a Built-in Anomaly Alert Rule.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_log_analytics_workspace" "example" {
+  name                = "example-law"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  sku                 = "PerGB2018"
+}
+
+resource "azurerm_security_insights_sentinel_onboarding" "example" {
+  resource_group_name          = azurerm_resource_group.example.name
+  workspace_name               = azurerm_log_analytics_workspace.example.name
+  customer_managed_key_enabled = false
+}
+
+data "azurerm_sentinel_alert_rule_anomaly" "example" {
+  log_analytics_workspace_id = azurerm_log_analytics_workspace.example.id
+  display_name               = "UEBA Anomalous Sign In"
+  depends_on                 = [azurerm_sentinel_log_analytics_workspace_onboarding.example]
+}
+
+resource "azurerm_sentinel_alert_rule_anomaly_built_in" "example" {
+  display_name               = "UEBA Anomalous Sign In"
+  log_analytics_workspace_id = azurerm_log_analytics_workspace.example.id
+  mode                       = "Production"
+  enabled                    = false
+}
```

Comment:
this example seems missing required values?

Reply:
as above, `name` is an optional value.